pages/glossary/focused.md

```diff
@@ -7,4 +7,6 @@ input_aspects:
   - CSS styling
 ---
 
-An element is _focused_ when that element matches the [`:focus` pseudo-class](https://drafts.csswg.org/selectors-4/#focus-pseudo).
+An element is said to be _focused_ when the element matches the [`:focus` pseudo-class](https://drafts.csswg.org/selectors-4/#focus-pseudo) uninterruptedly for a period of 5 seconds after a user interacted with the page.
+
+The 5 seconds time span is an arbitrary limit which is not included in WCAG. Giving the possibility of the focus state of elements being managed through scripts, testing the focused state of an element consistently would be impractical without a time limit.
```

Comment:
Interestingly:
* this rule doesn't really use the definition of "focused". Only in the assumption, which is OK to be less precise than the rest of the text. I'd say we can keep "focus" undefined in the Assumption without really breaking anything 🤔 
* the definition of "focusable" introduces a 1s limit, this one a 5s limit. They should probably be the same (otherwise something that redirects focus after 3s would be "focusable" but could never be "focused", which is at best misleading).
* "focused" is only used in [Element in sequential focus order has visible focus)(https://act-rules.github.io/rules/oj04fd). Since 2.4.7 does not care about the limit, this makes the rule a bit worse (but only creating false negatives, so it's OK).


I would argue in favour of leaving the definition of "focused" as it is and not define "focus" in the assumption. I think it is OK to have undefined terms in assumptions and leave them to "common knowledge".
(for example in [ARIA required context role](https://act-rules.github.io/rules/ff89c9#assumptions) we have an assumption that explicit roles are "appropriate" for their element without explaining fully what it means)

Reply:
Agree with suggestion.